static/scripts/calendar.js

```diff
@@ -143,8 +167,12 @@ $(document).ready(() => {
 				return false;
 			}
 			// personal event
-			const startDate = toMoment(event.start, calendar).format($t('format.dateTimeToPicker'));
-			const endDate = toMoment(event.end || event.start, calendar).format($t('format.dateTimeToPicker'));
+			const startDate = toMoment(event.start, calendar).format(
```

Comment:
Check your formater settings: if you do not change anything... the code should not be formated differently

Reply:
thanks for pointing it out, corrected my mistake.